### PR TITLE
Show claim items in subclaim basic info

### PR DESCRIPTION
**Pull request by falklinus**
# Jira Issue: [CLX-193]

## What?
- Don't use property selection for "Item", show claim items instead

## Why?
- Move away from property selection
- Use claim items

## Optional screenshots
image
image

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [x] Tested locally

[CLX-193]: https://hedvig.atlassian.net/browse/CLX-193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by vonElfvin** [comment]
Some updates:
image
image

File: src/portals/hope/features/claims/claim-details/ClaimItems/ClaimItem.tsx [hunk 1]
```diff
@@ -116,9 +119,18 @@ export const ClaimItem = ({
...
+                  contents={
+                    'Age ' +
+                    (dateOfOccurrence
+                      ? `${formatDistanceWithAccuracy(
+                          parseISO(item.purchaseDate),
+                          parseISO(dateOfOccurrence),
+                        )} ~ ${formatDistanceWithFraction(
+                          parseISO(item.purchaseDate),
+                          parseISO(dateOfOccurrence),
+                        )} years\n\n(compared to ${dateOfOccurrence})`
```
Comment (falklinus) [inline src/portals/hope/features/claims/claim-details/ClaimItems/ClaimItem.tsx:131]:
```suggestion
                        )} years\n(compared to ${dateOfOccurrence})`
```
Or does it look 🤮?

**Review by falklinus — COMMENTED** [review]
Bam!

**Comment by vonElfvin** [comment]
Update of cards:
image

File: src/portals/hope/features/claims/claim-details/ClaimItems/ClaimItem.tsx (hunk 1 above)
Comment (vonElfvin) [reply src/portals/hope/features/claims/claim-details/ClaimItems/ClaimItem.tsx:120]: I'd say they look about the same
image

**Review by vonElfvin — APPROVED** [review]
Me 🤝 Me

**Pull request merged**